pkgs/os-specific/linux/ply/default.nix

```
@@ -12,8 +12,8 @@ in stdenv.mkDerivation {
   src = fetchFromGitHub {
     owner = "iovisor";
     repo = "ply";
-    rev = "9e810b157ba079c32c430a7d4c6034826982056e";
-    sha256 = "15cp6iczawaqlhsa0af6i37zn5iq53kh6ya8s2hzd018yd7mhg50";
+    rev = "899afb0c35ba2191dd7aa21f13bc7fde2655c475";
```

I think we can now switch to tags:

```suggestion
    rev = version;
```